src/Workspaces/Core/Portable/SemanticModelWorkspaceService/SemanticModelWorkspaceServiceFactory.SemanticModelWorkspaceService.cs

```diff
@@ -21,19 +21,8 @@
 
 namespace Microsoft.CodeAnalysis.SemanticModelWorkspaceService
 {
-    [ExportWorkspaceServiceFactory(typeof(ISemanticModelService), ServiceLayer.Default), Shared]
```

Comment:
Split into a separate partial definition so the nested type can be in a separate file, and have different definitions in CodeStyle and Workspaces layers.